cypress/integration/core_opensearch_dashboards/opensearch_dashboards/apps/explore/04/traces_details.spec.js

```diff
@@ -0,0 +1,633 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  DATASOURCE_NAME,
+  TRACE_INDEX_PATTERN,
+  TRACE_TIME_FIELD,
+  TRACE_INDEX,
+} from '../../../../../../utils/apps/explore/constants';
+import { getRandomizedWorkspaceName } from '../../../../../../utils/apps/explore/shared';
+import { prepareTestSuite } from '../../../../../../utils/helpers';
+
+const workspaceName = getRandomizedWorkspaceName();
+
+const traceDetailsTestSuite = () => {
+  describe('Trace Details Interactive Tests', () => {
+    let traceUrl;
+
+    before(() => {
+      // Set up workspace and data source with traces
+      cy.explore.setupWorkspaceAndDataSourceWithTraces(workspaceName, [TRACE_INDEX]);
+
+      // Create index pattern for traces
+      cy.createWorkspaceIndexPatterns({
+        workspaceName: workspaceName,
+        indexPattern: TRACE_INDEX_PATTERN.replace('*', ''),
+        timefieldName: TRACE_TIME_FIELD,
+        indexPatternHasTimefield: true,
+        dataSource: DATASOURCE_NAME,
+        isEnhancement: true,
+        signalType: 'traces',
+      });
+
+      // Navigate to traces page and capture trace URL
+      cy.osd.navigateToWorkSpaceSpecificPage({
+        workspaceName: workspaceName,
+        page: 'explore/traces',
+        isEnhancement: true,
+      });
+
+      // Click on dataset selector to close Syntax options if blocking time-picker
```

Comment:
you dont need to do this way. there is logic so we don't have users having to click the dataset button every time. so we can mimic that behavior like this

```
cy.window().then((win) => {
  win.localStorage.setItem('hasSeenInfoBox_PPL', true);
});
```

this will be more consistent because if for some reason it doesn't find the datasetSelectButton then this test will fail on stuff that is unrelated to this test

Reply:
thanks yeah. i started doing that whenever i touch a file. that does this

Reply:
Added
cy.window().then((win) => {
  win.localStorage.setItem('hasSeenInfoBox_PPL', true);
});
And removed previous click item implementation as discussed.